### ISSUE TITLE
Exec Plugin to Capsule Plugin Mapping

### ISSUE THREAD

**Issue by kilemensi** [chrischristo/capsule-maven-plugin#30]
Hi @chrischristo

From the `exec:java` plugin [Java Goal] (http://www.mojohaus.org/exec-maven-plugin/usage.html) usage page, `arguments` are supposed to be mapped to *application* arguments and not *JVM* arguments as in the current implementation.

I'm raising this issue because by doing this way, if I use `mvn exec:java` to run my application it runs fine but when I want to run the application through the final capsule jar, it gives me an error caused by `arguments` which have been mapped to `JVM-args` being specified first in the command line before the `appClass` making java think the first `argument` is the class name.

TLDR:
From `exec` configuration I expect the final capsule command to be: `java <systemProperties/> <mainClass/> <arguments/>` but the mapping or `arguments` to `JVM-args` causes the command to be `java <arguments/> <systemProperties/> <mainClass/>` causing program exec failure.

EDIT
I believe exec `arguments` should be mapped to the `Args` manifest attribute.

**Comment by chrischristo** [comment]
Ah ok thanks for reporting! Away at the moment, will fix this around oct5

**Comment by chrischristo** [comment]
Very good spot! Will be fixed in upcoming v1.0.

**Comment by chrischristo** [comment]
[Fixed in 1.0.0](https://github.com/chrischristo/capsule-maven-plugin/releases/tag/capsule-maven-plugin-1.0.0).
